LMS/library.py

```diff
@@ -0,0 +1,97 @@
+from typing import List, LiteralString, Any
+
+from book import Book
+
+
+class Library:
+    """Class Library would LIST, LEND, RETURN or ADD book(s) to the library as per the input by the user"""
+
+    def __init__(self, listOfBooks: List[Book]):
+        self.availableBooks = listOfBooks
+        self.borrowed_books = []
+
+    def displayAvailableBooks(self):
+        for book in self.availableBooks:
+            if self.is_book_available(book):
+                print(f'Book: {book.name}')
```

Comment:
Ideally Book class can have a method called pretty_print. That way the book itself knows whats the best way to print it.